packs/data/iconics.db/quinn-level-5.json

```diff
@@ -3949,4 +3949,4 @@
         }
     },
     "type": "character"
-}
+}
```

Comment:
Restore this line break at once!

Reply:
Put that shit in the Prettier config file. :wink: